Podfile

```diff
@@ -17,6 +17,12 @@ target 'UnitTests' do
   pod 'AEPCore'
 end
 
+target 'UpstreamIntegrationTests' do
+  pod 'AEPCore'
+  pod 'AEPEdgeIdentity'
+  pod 'AEPEdgeConsent'
+end
```

Comment:
I believe you need to include `pod 'AEPEdge', :path => './AEPEdge.podspec'` here so the Edge Identity or Edge Consent projects don't pull in the released AEPEdge Pod dependency.